### PR TITLE
Delete some block-building test helpers that have been replaced with chain.Builder

### PR DESCRIPTION
**Pull request by anorth**
cc @hannahhoward

### CODE REVIEW

File: sampling/chain_randomness_test.go [hunk 1]
```diff
@@ -20,80 +21,92 @@ func TestSamplingChainRandomness(t *testing.T) {
...
 		// Ok when the chain is just sufficiently long.
-		r, err := sampling.SampleChainRandomness(types.NewBlockHeight(uint64(19)), chain)
+		r, err := sampling.SampleChainRandomness(types.NewBlockHeight(uint64(19)), ch)
 		assert.NoError(t, err)
 		assert.Equal(t, []byte(strconv.Itoa(16)), r)
 	})
 
 	t.Run("falls back to genesis block", func(t *testing.T) {
-		chain := testhelpers.RequireTipSetChain(t, 5)
+		_, ch := makeChain(t, 6)
```
Comment (frrist) [inline sampling/chain_randomness_test.go]: any reason we are increasing this value (5 -> 6)?

File: sampling/chain_randomness_test.go [hunk 2]
```diff
@@ -20,80 +21,92 @@ func TestSamplingChainRandomness(t *testing.T) {
...
 
-		r, err = sampling.SampleChainRandomness(types.NewBlockHeight(uint64(10)), chain)
+		r, err = sampling.SampleChainRandomness(types.NewBlockHeight(uint64(10)), ch)
 		assert.NoError(t, err)
 		assert.Equal(t, []byte(strconv.Itoa(7)), r)
 	})
 
 	t.Run("skips missing tipsets", func(t *testing.T) {
-		chain := testhelpers.RequireTipSetChain(t, 20)
+		builder, ch := makeChain(t, 21)
```
Comment (frrist) [inline sampling/chain_randomness_test.go]: Why is this change needed? 20 -> 21

**Comment by ZenGround0** [comment]
Looks like some ints still need to become uints but LGTM in general

File: sampling/chain_randomness_test.go (hunk 1 above)
Comment (anorth) [reply sampling/chain_randomness_test.go]: Yes, `RequireTipSetChain` misleadingly created a chain one longer than that requested, i.e. the genesis plus the requested count.
